### PR TITLE
Edit + explain id formatting

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [ ] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

Yes, this is related to https://github.com/thegetty/quire/issues/869 as reported by @cbutcosk

### Please describe the goal of this pull request and the changes that were made.

With the switch to 11ty, `id`s have become more sensitive. If they include periods, upper case letters, underscores, etc. they will cause the preview to break. The documentation has been updated to reflect this change.

I also took this as an opportunity to made additional edits/clarifications. 

### Additional Comments

Thank you @cbutcosk for reporting this inconsistency with the docs!

### CODE REVIEW

File: content/docs-v1/figure-images.md [hunk 1]
```diff
@@ -25,22 +25,22 @@ Quire does not require a specific image file format or size, but we have some re
...
+```yaml
+- id: "portrait-of-still"
   src: "portrait-of-still.jpg"
 ```
 
-Other available attributes are `alt`, `caption`, `credit`, `media_id`, `media_type`, `aspect_ratio`, and `label`.
+The `id` is an assigned value that is used to reference the image when using a shortcode. The `id` can be anything as long as it is **all lowercase with no spaces**. You can separate words through the use of dashes (-), as in the two examples above. The use of uppercase letters, periods, spaces, or underscores will cause the preview to break. It is helpful to keep the `id` short, otherwise adding shortcodes can become cumbersome.
 
-{{< q-class "box warning" >}}
+The `src` points to where you have saved your image, with the default being `/_assets/images` unless otherwise noted in the `config.yaml`. You can also create subfolders in the `/_assets/images` directory which then need to be included in the `src`. For example, if you created a folder inside the `/_assets/images` directory called "catalogue" and put the `portrait-of-still.jpg` image in that folder, then the `src` would be `/catalogue/portrait-of-still.jpg`.
```
Comment (geealbers) [inline content/docs-v1/figure-images.md:41]: There shouldn't be a slash at the beginning: `catalogue/portrait-of-still.jpg`

File: content/docs-v1/figure-images.md [hunk 2]
```diff
@@ -51,33 +51,28 @@ Unless a figure image is purely decorative, it should also include an alternate
 For example, the YAML for the figure above might look like this:
 
 ```YAML
-- id: "5.5"
-  src: rococo_cat-1.jpg
+- id: "cat-1"
+  label: "Cat. 1"
+  src: /catalogue/rococo_cat-1.jpg
```
Comment (geealbers) [inline content/docs-v1/figure-images.md:56]: Remove slash: `src: catalogue/rococo_cat-1.jpg`

File: content/docs-v1/figure-images.md [hunk 3]
```diff
@@ -160,7 +153,7 @@ src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/80
 You would then grab `809104705` and add that as a `media_id` to the YAML for this audio file. Similar to the videos you will also want to include a `poster` attribute as a fallback for your PDF and EPUB versions of the publication.
 
 ```YAML
-- id: "cva_podcast"
+- id: "cva-podcast"
   poster: /screenshots/cva_podcast.png
```
Comment (geealbers) [inline content/docs-v1/figure-images.md:157]: remove slash

**Review by geealbers — CHANGES_REQUESTED** [review]
Looks good. Just need to remove the erroneous preceding slashes we had in there for the image `src`

**Comment by Erin-Cecele** [comment]
Thank you @geealbers. The change has been made

**Review by geealbers — APPROVED** [review]

**Pull request merged**